openpype/hosts/blender/api/lib.py

```diff
@@ -25,6 +26,47 @@
 
 log = Logger.get_logger(__name__)
 
+
+def get_root_containers_from_datablocks(
+    datablocks: List[bpy.types.ID],
+) -> List[bpy.types.ID]:
+    """Get root containers from a list of datablocks.
+    
+    Args:
+        datablocks (List[bpy.types.ID]): Datablocks to get root containers from.
+
+    Returns:
+        List[bpy.types.ID]: Root containers.
+    """
+    openpype_containers = bpy.context.window_manager.openpype_containers
+    if not openpype_containers:
+        update_scene_containers()
+
+    # Get all avalon data from openpype_containers as list of dicts
+    all_containers_avalon_data = [
+        c[AVALON_PROPERTY].to_dict()
+        for c in bpy.context.window_manager.openpype_containers
```

Comment:
Why storing openpype_containers in a variable if you don't use it?

Reply:
Forgot to refactor that